packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -33,7 +33,7 @@ export const DIR_NODE_CLASS = 'theia-DirNode';
 export const FILE_STAT_ICON_CLASS = 'theia-FileStatIcon';
 
 @injectable()
-export class FileTreeWidget extends TreeWidget {
+export class FileTreeWidget extends TreeViewWelcomeWidget {
```

Comment:
Can we enable the welcome-support in all other trees as well? It isn't FS specific. What do you think, @akosyakov?